docs/modules/ROOT/pages/how-to/monitor-http-endpoints.adoc

```diff
@@ -0,0 +1,262 @@
+= Monitor HTTP(S) Endpoints Using Blackbox Exporter
+
+== Prerequisites
+
+* Command Line Access to APPUiO Cloud
+* Helm 3, https://helm.sh/
+
+== Install Blackbox Exporter
+
+. Set the namespace for the Blackbox Exporter
++
+[source,bash]
+----
+APP_NAMESPACE="my-app" <1>
+EXPORTER_NAMESPACE="my-org-blackbox-exporter" <2>
+----
+<1> The namespace of your application.
+<2> The exporter can be installed in the same namespace as your application or in a separate namespace.
+You might need additional network policies if installing the exporter in a separate namespace.
+See the <<tips-and-tricks>> section for more information.
```

Comment:
There's no networkpolicy shown in tips-and-tricks. So there's no way to end up with a working setup by following this guide using the default setup of having the blackbox-exporter in a separate namespace.